2.6.1/pg10/Dockerfile

```diff
@@ -1,16 +1,46 @@
-FROM pgrouting/postgres:pg10
+FROM mdillon/postgis:10
```

Comment:
Discussing this change in main thread: https://github.com/pgRouting/docker-pgrouting/pull/12#issuecomment-509186633